packages/vscode-extension/src/common/Project.ts

```diff
@@ -127,7 +127,7 @@ export interface ProjectInterface {
   updatePreviewZoomLevel(zoom: ZoomLevelType): Promise<void>;
 
   getDeviceSettings(): Promise<DeviceSettings>;
-  updateDeviceSettings(deviceSettings: DeviceSettings): Promise<void>;
+  updateDeviceSettings(deviceSettings: DeviceSettings, sendToDevice?: boolean): Promise<void>;
```

Comment:
Why do we need this extra argument? It appears like we should be able to tell which setting changed based on the deviceSettings object, and it should be up to the implementation to decide when the restart is needed and not up to the UI.